translations/core/POTFILES.in

```diff
@@ -16,48 +16,48 @@ client/connectdlg_common.cpp
 client/control.cpp
 client/editor.cpp
 client/global_worklist.cpp
-client/gui-qt/canvas.cpp
-client/gui-qt/chatline.cpp
-client/gui-qt/citydlg.cpp
-client/gui-qt/cityrep.cpp
-client/gui-qt/dialogs.cpp
-client/gui-qt/diplodlg.cpp
-client/gui-qt/economyreport.cpp
-client/gui-qt/endgamereport.cpp
-client/gui-qt/fc_client.cpp
-client/gui-qt/fonts.cpp
-client/gui-qt/gotodlg.cpp
-client/gui-qt/gui_main.cpp
-client/gui-qt/helpdlg.cpp
-client/gui-qt/hudwidget.cpp
-client/gui-qt/icons.cpp
-client/gui-qt/idlecallback.cpp
-client/gui-qt/luaconsole.cpp
-client/gui-qt/mapctrl.cpp
-client/gui-qt/mapview.cpp
-client/gui-qt/menu.cpp
-client/gui-qt/messageoptions.cpp
-client/gui-qt/messagewin.cpp
-client/gui-qt/notifyreport.cpp
-client/gui-qt/optiondlg.cpp
-client/gui-qt/page_main.cpp
-client/gui-qt/page_network.cpp
-client/gui-qt/page_pregame.cpp
-client/gui-qt/page_scenario.cpp
-client/gui-qt/plrdlg.cpp
-client/gui-qt/pregameoptions.cpp
-client/gui-qt/ratesdlg.cpp
-client/gui-qt/sciencedlg.cpp
-client/gui-qt/shortcuts.cpp
-client/gui-qt/spaceshipdlg.cpp
-client/gui-qt/themes.cpp
-client/gui-qt/tooltips.cpp
-client/gui-qt/top_bar.cpp
-client/gui-qt/tradecalculation.cpp
-client/gui-qt/unitreport.cpp
-client/gui-qt/unitselect.cpp
-client/gui-qt/voteinfo_bar.cpp
-client/gui-qt/widgetdecorations.cpp
+client/canvas.cpp
+client/chatline.cpp
+client/citydlg.cpp
+client/cityrep.cpp
+client/dialogs.cpp
+client/diplodlg.cpp
+client/economyreport.cpp
+client/endgamereport.cpp
+client/fc_client.cpp
+client/fonts.cpp
+client/gotodlg.cpp
+client/gui_main.cpp
+client/helpdlg.cpp
+client/hudwidget.cpp
+client/icons.cpp
+client/idlecallback.cpp
+client/luaconsole.cpp
+client/mapctrl.cpp
+client/mapview.cpp
+client/menu.cpp
+client/messageoptions.cpp
+client/messagewin.cpp
+client/notifyreport.cpp
+client/optiondlg.cpp
+client/page_main.cpp
+client/page_network.cpp
+client/page_pregame.cpp
+client/page_scenario.cpp
+client/plrdlg.cpp
+client/pregameoptions.cpp
+client/ratesdlg.cpp
+client/sciencedlg.cpp
+client/shortcuts.cpp
+client/spaceshipdlg.cpp
+client/themes.cpp
+client/tooltips.cpp
+client/top_bar.cpp
+client/tradecalculation.cpp
+client/unitreport.cpp
+client/unitselect.cpp
+client/voteinfo_bar.cpp
+client/widgetdecorations.cpp
```

Comment:
Hadn't seen it last night, can you sort it alphabetically?

Reply:
> Hadn't seen it last night, can you sort it alphabetically?

Of course